InternshipBE/Tests/Tests.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
yep, choose a test tool like XUnit or NUnit and create project  
![image](https://user-images.githubusercontent.com/71830071/137166600-1742413d-f89e-42cf-a869-7505233c1965.png)